build/Targets/Settings.props

```diff
@@ -13,6 +13,7 @@
     <PackageProjectUrl>https://github.com/dotnet/roslyn</PackageProjectUrl>
     <PackageTags>Roslyn CodeAnalysis Compiler CSharp VB VisualBasic Parser Scanner Lexer Emit CodeGeneration Metadata IL Compilation Scripting Syntax Semantics</PackageTags>
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
+    <NoWarn>$(NoWarn);NU5125</NoWarn>
```

Comment:
Want to add a comment pointing to the bug tracking the removal?

Reply:
Good call.